src/components/DraftOrdersTab.tsx

```diff
@@ -41,7 +42,20 @@ export function DraftOrdersTab() {
           <TableHeader className="bg-background">
             <TableRow>
               <TableHead className="rounded-tl-md">
-                <span className="sr-only">Select</span>
+                <Checkbox
+                  checked={
+                    selectedIds.length === draftOrders.length &&
+                    !!draftOrders.length
```

Comment:
what happens when the user manually deselects one of the orders?

Reply:
It also deselects the select all checkbox